src/parser/parser_support.c

```diff
@@ -1201,6 +1201,40 @@ pt_is_inst_or_orderby_num_node (PARSER_CONTEXT * parser, PT_NODE * tree, void *a
   return tree;
 }
 
+/*
+ * pt_is_inst_or_inst_num_node () -
+ *   return:
+ *   parser(in):
+ *   tree(in):
+ *   arg(in/out): true if node is an INST_NUM or ORDERBY_NUM or GROUPBY_NUM expression node
+ *   continue_walk(in/out):
+ */
+PT_NODE *
+pt_is_inst_or_inst_num_node (PARSER_CONTEXT * parser, PT_NODE * tree, void *arg, int *continue_walk)
+{
+  bool *has_inst_orderby_num = (bool *) arg;
+
+  if (PT_IS_INSTNUM (tree) || PT_IS_ORDERBYNUM (tree) || PT_IS_GROUPBYNUM (tree))
+    {
+      *has_inst_orderby_num = true;
+    }
+
+  if (*has_inst_orderby_num)
+    {
+      *continue_walk = PT_STOP_WALK;
+    }
```

Comment:
It seems that it can be made as follows by removing the has_inst_orderby_num variable and reducing the substitution statement.

```
if ( (bool *) arg || PT_IS_INSTNUM (tree) || PT_IS_ORDERBYNUM (tree) || PT_IS_GROUPBYNUM (tree))
    {
        *continue_walk = PT_STOP_WALK;
        return tree;
    }

```

Reply:
1. has_inst_orderby_num is return value.
2. The reason why the else if statement below is needed is to not walking for subqueries included in the node.
```
else if (PT_IS_QUERY_NODE_TYPE (tree->node_type))
```

Reply:
OK,